core/templates/domain/exploration/RecordedVoiceoversObjectFactory.ts

```diff
@@ -16,22 +16,34 @@
  * @fileoverview Factory for creating new frontend instances of
  * RecordedVoiceovers domain objects.
  */
-
-import { Injectable } from '@angular/core';
+export interface IRecordedVoiceOverBackendDict {
+  'voiceovers_mapping': {
+    [propName: string]: {
```

Comment:
What does this mean? 

voiceovers_mapping` can have any number of objects where each object can have any number of properties each of `IVoiceoverDict`.

Reply:
@bansalnitish 
yes it has the same meaning as you mention above
this can be clear from this too
````
   voiceovers_mapping: {
      content: {
        en: {
          filename: 'filename1.mp3',
          file_size_bytes: 100000,
          needs_update: false,
          duration_secs: 10.0
        },
        hi: {
          filename: 'filename2.mp3',
          file_size_bytes: 11000,
          needs_update: false,
          duration_secs: 0.11
        }
      },
      default_outcome: {
        en: {
          filename: 'filename3.mp3',
          file_size_bytes: 3000,
          needs_update: false,
          duration_secs: 0.33
        },
        hi: {
          filename: 'filename4.mp3',
          file_size_bytes: 5000,
          needs_update: false,
          duration_secs: 0.5
        }
      },
}
````
thanks

Reply:
I see -- I was worried that type is a bit generic here but given the example that you've shown it looks like the type is fine :). Thanks for clarifying, I'll just take a complete look on the PR now.